data/pages-content/pyladies-workshop.md

```diff
@@ -0,0 +1,22 @@
+## About PyLadies
+PyLadies is an international mentorship group with focus on helping under-represented developers become active participants and leaders in the Python open-source community.
+Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
+
+## About Workshop 
+Are you a woman or a girl interested in programming but unsure where to begin? Don't worry, we have the perfect workshop for you, designed to be welcoming and accessible to complete beginners!
```

Comment:
I'd switch the mentions: PyLadies explicitly say they focus on more women in tech, but we'd like the workshop to be open to all under-represented folks.
The key message for the banner would then consist of 3 parts: no money involved, beginners, primarily for a minority group. I'll try to come up with some ideas with a various degree of silliness.

I also much like the "Don't worry, we have the perfect workshop for you, designed to be welcoming and accessible even if you don't have any prior programming experience!" sentence.

Reply:
Okay, I've switched the mentions, check it out. 

About banner - how about this:
PyLadies Snake Workshop
Join us on 17 July 2023
Beginner-friendly, empowering diversity!

What do you think? I'd like to keep it to three lines, because if there are 4 there's too much of text on the left side and it doesn't look good. If you have any other ideas, let me know 🙂

Reply:
I love it. One 'cosmetic' feedback I have: I'd rather have "PyLadies" in the banner in one color and "Snake Workshop" in another. I can't get rid of the "PyLadies Snake" impression. Thank you for creating the banner, that's a great idea!

Reply:
I've changed the web banner colors, you can see the preview here: https://ep2023-hy1tmey60-europython-society.vercel.app/pyladies-workshop Let me know if you have any other feedback, if not then I'll wait for the Google forms link and that should be the last thing when it comes to web.

As for the social media banner, I've changed it as well, check it out and let me know what do you think. 🙂 If it looks good for you, then I'll upload it to the drive with other banners.
![image](https://user-images.githubusercontent.com/38294198/235964983-a6141588-36ba-47db-af4e-8bdf3748d84e.png)

Reply:
> As for the social media banner, I've changed it as well, check it out and let me know what do you think. slightly_smiling_face If it looks good for you, then I'll upload it to the drive with other banners.

I like it! One last remark: could you move the last line to be aligned with the 1st and 3rd?

Reply:
> How about adding this screenshot just after the main header before "About PyLadies"?

The sreenshot itself is not good for advertising. The graphics aren't the point of the workshop, and intentionally have a simple “DYI” look, so for a “first impression” or a representation of what we'll make, it's underwhelming. Feel free to use it for the palette or decorations though.

Reply:
I've updated the banner:
![image](https://user-images.githubusercontent.com/38294198/236261086-91c83f5c-9226-448a-a1be-0cd284a8af45.png)


As for adding the snake logo on the webpage - I'm not a designer, but I feel like these colors are not compatible with the rest of the page, even if we make it smaller, as:
1) there's different shade of red in the logo and on the button "Code of conduct" - they're too similar yet different, 
2) same goes for deep blue in EuPy logo and header, and black background
3) green has high saturation, same as deep blue in the banner and red button on the right side - in this combination I feel like it's too much. 

![image](https://user-images.githubusercontent.com/38294198/236261961-51b1be37-7581-4d32-8229-ffdb1420f347.png)

Reply:
I love this version of the banner!

Reply:
Great. @hypha could you please upload it then to https://drive.google.com/drive/folders/1cmOLAsNUSO2L8FCYmH5TZS5gon79buPY, I can send you it by email, or could you give me rights to upload it myself? Thanks

Reply:
I've given you access @clytaemnestra. Feel free to put the banner and/or social card img there! 

You might also want to push the social card here which I hope would make the card automatically render when we post the web page: https://github.com/EuroPython/website/tree/809bf49bce49175ee095a935195c1473eaa83ed7/social-cards

Reply:
Thanks, I've uploaded the banner and pushed it to the folder with other banners. In that case I'm closing this thread, as I've addressed all your remarks. Thanks you folks for your feedback, I think it looks great 🙂  

The final thing left to do is update the forms link, which I'll do after we finalize it and we can merge it 🙂 

When it comes to social event, I'll create a new MR so not to have it all in this one. As we need less preparation for it, I'll start working on it later - probably at the end of the month, as I still need to check out few venues and confirm it.

Reply:
hey @clytaemnestra apologies for opening this again. Would you mind putting the title, subtitle, and `social_card: ep2023-social-card-banner-snake-workshop.png` at the top of the markdown file? This is to generate the metatags and make sure that the social card automatically renders. Here is the Trans*Code example:
https://github.com/EuroPython/website/blob/e4ad8aff7777ec56cac6fdef922a8b242bc8eb79/data/pages-content/trans_code.md?plain=1#L1-L7

Really sorry for not catching it earlier!

Reply:
Looks fantastic! Thank you so much @clytaemnestra! Feel free to resolve the thread!